src/plugins/discover_legacy/public/plugin.ts

```diff
@@ -318,6 +320,25 @@ export class DiscoverPlugin
         if (!this.initializeInnerAngular) {
```

Comment:
Are we in a state to remove this or is the legacy still using Angular?

Reply:
The legacy still uses angular